app/src/test/java/org/simple/clinic/editpatient/PatientEditScreenValidationUsingMockDateValidator.kt

```diff
@@ -61,22 +62,19 @@ class PatientEditScreenValidationUsingMockDateValidator {
 
   private val utcClock: TestUtcClock = TestUtcClock()
   private val dateOfBirthFormat = DateTimeFormatter.ofPattern("dd/MM/yyyy", Locale.ENGLISH)
-  private lateinit var dobValidator: UserInputDateValidator
+  private val nextYear = LocalDate.now().year + 1
```

Comment:
Added a task to improve the `UserInputDateValidator` so that we do not need to compute the next year in tests. https://www.pivotaltracker.com/story/show/168775283